File: app/src/at-myhuynh/java/asiantech/internship/summer/JavaTrainning/src/ex19/SumNumberPrime.java
package ex19;

import common.Common;

public class SumNumberPrime {
	public static void main(String[] args) {
		int n = 14;
		
		for(int i = 10000; i < 100000; i++){
			if(Common.isPrime(i) && Common.totalOfNumber(i) == n){
				System.out.println(i);
			}
		}

	}
}

File: app/src/at-myhuynh/java/asiantech/internship/summer/kotlin/ex19/SumNumberPrime.kt
package asiantech.internship.summer.kotlin.ex19

import asiantech.internship.summer.kotlin.common.Common
import asiantech.internship.summer.kotlin.common.isPrime
import asiantech.internship.summer.kotlin.common.totalOfNumber

object SumNumberPrime {
    @JvmStatic
    fun main(args: Array<String>) {
        val n = Common.input("Nhập vào số n:")
        (10000..99999).filter { it.isPrime() && it.totalOfNumber() == n }.forEach(::println)
    }
}
